server/integration_tests/test_data/international/query_3/chart_config.json

```diff
@@ -93,35 +93,6 @@
             ],
             "title": "Nominal GDP Per Capita"
           },
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "sdg/NY_GDP_PCAP"
```

Comment:
Why do we lose this for GDP query?

Reply:
with growth rate and per capita,  this seems not that close to the query anymore